internal/vm/dir.go

```diff
@@ -49,7 +49,8 @@ func shimDir(varRunDir, namespace, vmID string) (Dir, error) {
 		return "", errors.Wrapf(err, "failed evaluating any symlinks in path %q", varRunDir)
 	}
 
-	return Dir(filepath.Join(resolvedVarRunDir, namespace, vmID)), nil
+	n := namespace + "-" + vmID
+	return Dir(filepath.Join(resolvedVarRunDir, n, vmID)), nil
```

Comment:
Let's say we have (namespace=`aaa`, vmID=`bbb-ccc`) and (namespace=`aaa-bbb`, vmID=`ccc`), does it work?

Reply:
It pass all unit tests, I just add these two cases into `dir_test.go`. I am not sure if it still works if both of them try to remove the namespace dir, they share the same namespace dir.

Reply:
Yeah. I think it wouldn't work.